### PR TITLE
Initial productionization

### LINKED ISSUE #4: Fence output
We should try to fence the output in some way, both to make it easier to find (with `less/grep`) and to help separate it from the main application output.

### LINKED ISSUE #5: Output to file

### LINKED ISSUE #6: Consider output format
Currently we're printing raw JSON, which makes sense for machine consumption but looks very out of place in a log dump, especially if it's broken up per-line by some log aggregation system.

### LINKED ISSUE #7: Loop

### PR DESCRIPTION
**Pull request by nightkr**
Various changes to make it more suitable for integrating into the SDP, see https://github.com/stackabletech/issues/issues/527

For an example integration, see https://github.com/stackabletech/zookeeper-operator/pull/881

Fixes #4, fixes #5, fixes #6, fixes #7

The output now looks something like this in the terminal log:

image

or like this in a vector target like OpenSearch:

image

The JSON output is preserved, and can be enabled with the `--output` option. Logs are dumped as collection happens, which should also help users better understand why/whether it's running slow.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -1,158 +1,74 @@
...
+
+use crate::system_information::SystemInformation;
+use std::time::Instant;
+
+const APP_NAME: &str = "containerdebug";
+
+/// Collects and prints helpful debugging information about the environment that it is running in.
+#[derive(clap::Parser)]
+struct Opts {
+    /// Loop every <DURATION>, instead of shutting down once completed (default <DURATION>: 1m)
```
Comment (adwk67) [inline src/main.rs]: FWIW this raises a warning about an incorrectly closed html tag.

**Review by adwk67 — APPROVED** [review]
Very nice! Tested this with the associated zookeeper draft PR and logs appear in the terminal log window and also in json format.

File: src/main.rs (hunk 1 above)
Comment (nightkr) [reply src/main.rs]: Oh joy >.<.

Comment (nightkr) [reply src/main.rs]: 8999b1597030871635115ae7ddebd9e6aaab0dce

**Review by adwk67 — APPROVED** [review]

**Pull request merged**